lib/matplotlib/cbook.py

```diff
@@ -361,9 +361,11 @@ class _BoundMethodProxy(object):
     Minor bugfixes by Michael Droettboom
     '''
     def __init__(self, cb):
+        self._hash = hash(cb)
+        self._callbacks = []
```

Comment:
Almost, they get called when the object (of the method we proxy), gets destroyed, if we proxy a object method, otherwise it will never get called (then again I don't think a class method nor a function would get destroyed).

So basically I use this as a way to tell those that hardref'ed to this ``_BoundMethodProxy`` "I have died inside, so let me go so that I myself can die (and get GC'ed)"